src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -79,6 +79,14 @@ public override string Name
         public abstract AssemblyIdentity Identity { get; }
 
         /// <summary>
+        /// Assembly version as specified in source, with wildcards represented by <see cref="ushort.MaxValue"/>.
+        /// For example, 
+        ///   AssemblyVersion("1.2.*") is represented as 1.2.65535.65535,
```

Comment:
It's an error. All version parts in AssemblyVersion attribute have to be < 65535.